packages/react/package.json

```diff
@@ -54,6 +54,7 @@
     }
   ],
   "dependencies": {
+    "@storybook/cli": "^7.0.21",
     "@raisins/core": "^1.1.3",
     "@raisins/schema": "^1.0.0",
     "bunshi": "^2.1.4",
```

Comment:
We need to downgrade bunshi to an earlier version, raisins has a bug with bunshi 2.1

Reply:
Addressed in [this PR](https://github.com/saasquatch/raisins/pull/17). Will make sure to merge that one before this.